Blink/SpaceController.swift

```diff
@@ -213,6 +213,13 @@ class SpaceController: UIViewController {
     _removeCurrentSpace()
   }
   
+  /**
+    Sends `Ctrl + L` sequence when `Cmd + K` is pressed. Mimics the iTerm's clean screen behavior. Behaves the same as `Ctrl + L`.
+   */
+  @objc private func _clearCurrentTerminal() {
+    currentDevice?.view.webView.reportPress(.control, keyId: "76:0")
```

Comment:
I think we should add `l` to KeyCode enum. So instead of magic `76:0` it will be KeyCode.l.keyId

Reply:
Actually we don’t need that code at all.

You need just add `0C` Hex here https://github.com/blinksh/blink/blob/449882bf526cd1aeefc1f8153e0ae22b766553dc/KB/Native/Model/KeyBindingAction.swift#L167